code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -114,7 +114,7 @@ Please contact me on #coderbus IRC. ~Carn x
 //Human Overlays Indexes/////////
 #define MUTATIONS_LAYER			1
 #define SKIN_LAYER				2
-#define DAMAGE_LAYER			3
+#define D_LAYER					3
```

Comment:
these layer names are less descriptive, not a good change.

If theres a conflict with something else, make them more descriptive to clarify the use

Reply:
Ready